src/CommunityToolkit.Maui.MediaElement/Services/MediaControlsService.android.cs

```diff
@@ -58,7 +58,7 @@ public override StartCommandResult OnStartCommand([NotNull] Intent? intent, Star
 			{
 				foreach (var exception in t.Exception.InnerExceptions)
 				{
-					System.Diagnostics.Trace.WriteLine($"[error] {exception}, {exception.Message}");
+					System.Diagnostics.Trace.TraceError("[error] {0}, exception");
```

Comment:
Please do not use `TraceError` in the library. It causes crash reporting tools like sentry.io to report it as an unhandled exception. Leading to our library causing erroneous bug reports for our users.